sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -408,7 +411,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
    *
    * @param <T> the type of values written to the sink.
    */
-  public abstract static class FileBasedWriteOperation<T> implements Serializable {
+  public abstract static class WriteOperation<T> implements Serializable {
```

Comment:
Not sure I understand - this renaming is already a separate commit, that I believe contains nothing else.